doc/rst/tools/chpldoc/chpldoc.rst

```diff
@@ -71,7 +71,7 @@ elsewhere, use the chpldoc command directory. For example::
 
 By default, documentation created by chpldoc will end up in a subdirectory of
 the current working directory named 'docs/'. You can specify a different
-directory name using the --docs-dir option.
+directory name using the --output-dir option.
```

Comment:
While here, I think it would be good to wrap the flag in "`", like ``$CHPL_HOME/examples/primers/`` is below.  Otherwise, this looks good to me!

Reply:
I was thinking the same. Will do that now.

Reply:
I have made the suggested changes.